packages/common/src/hooks/useSavedCollections.ts

```diff
@@ -1,84 +1,92 @@
-import { useCallback, useEffect, useState } from 'react'
+import { useCallback, useMemo } from 'react'
 
 import { useDispatch, useSelector } from 'react-redux'
 
+import { ID } from 'models/Identifiers'
 import { Status } from 'models/Status'
+import { CommonState } from 'store/index'
+import { getFetchedCollectionIds } from 'store/saved-collections/selectors'
 
-import { accountActions } from '../store/account'
 import {
+  CollectionType,
   savedCollectionsActions,
   savedCollectionsSelectors
 } from '../store/saved-collections'
 
-const { fetchSavedPlaylists } = accountActions
 const { fetchCollections } = savedCollectionsActions
 
-const {
-  getAccountAlbums,
-  getSavedAlbumsState,
-  getFetchedAlbumsWithDetails,
-  getAccountPlaylists
-} = savedCollectionsSelectors
+const { getAccountAlbums, getSavedCollectionsState, getAccountPlaylists } =
+  savedCollectionsSelectors
 
 const DEFAULT_PAGE_SIZE = 50
 
 export function useSavedAlbums() {
   return useSelector(getAccountAlbums)
 }
 
-/* TODO: Handle filtering
- * Option 1: This hook takes the list of album ids to fetch and computes the unfetched
- * based on that.
- * Option 2: Bake filter into selectors which drive this. Downside: Can't use this in multiple places...
- */
-type UseSavedAlbumDetailsConfig = {
+export function useSavedPlaylists() {
+  return useSelector(getAccountPlaylists)
+}
+
```

Comment:
After spending a while staring at this, I think it would be nice to call these hooks something that indicates that they're fetching the abbreviated `AccountCollection` that essentially just has the IDs, which is then to be fetched later as a full `Collection`. If I just look at the callsites for these and the fetching hook below, it feels like we're selecting a `Collection` out of the store, and then re-fetching it for some reason. Maybe `useAccountAlbums` - eventually would be great to rename this type into something more descriptive (`CollectionMetadata`? `CollectionPreview`? hmm), but probably avoid for now :D